### ISSUE TITLE
batch.download.daymet() error

### ISSUE THREAD

**Issue by bradleytbush** [bluegreen-labs/daymetr#4]
When attempting to call the code: 

batch.download.daymet(file_location='TestDoc.csv',start_yr=2011,end_yr=2015,internal=FALSE)

from an R script in RStudio (Version 0.99.903) the following error code is generated:

Error: could not find function "batch.download.daymet"

The newest version of DaymetR (downloaded 3 Oct 2016 10am AZ time) has been downloaded and loaded using the library() command. Important to note also is that when I run the code:

daymetr::batch.download.daymet(file_location='TestDoc.csv',start_yr=2011,end_yr=2015,internal=FALSE)

from the RStudio command window the following error code is generated:

Error in download.daymet(site = site, lat = lat, lon = lon, start_yr = start_yr,  : 
  You requested data is outside DAYMET coverage,
           the file is empty --> check coordinates!

The coordinates used were:   40.22458   -88.85675 (LAT/LON) which worked with a previous version of DaymetR and is not outside DAYMET coverage.

Attached is the TestDoc I used in my request code (converted from .csv to .xlsx for GitHub upload)
[TestDoc.xlsx](https://github.com/khufkens/daymetr/files/506808/TestDoc.xlsx)

**Comment by khufkens** [comment]
Works on Ubuntu and OSX, check all dependencies and update those. Check your formatting of the arguments. I can not provided support for Windows as I do not have access to such system. But it should work, given my tests.

> library("daymetr", lib.loc="~/R/x86_64-pc-linux-gnu-library/3.3")
> download.daymet()
> Downloading DAYMET data for: Daymet at 36.0133/-84.2625 latitude/longitude !
> Done !
> batch.download.daymet("Downloads/TestDoc.csv",start_yr = 1999)
> Downloading DAYMET data for: 48369 at 40.22458/-88.85675 latitude/longitude !
> Done !
> Downloading DAYMET data for: 48372 at 40.22458/-88.85675 latitude/longitude !
> Done !
> Downloading DAYMET data for: 48375 at 40.21809/-88.92174 latitude/longitude !
> Done !
> download.daymet(site="bla",lat=40.22458,lon=-88.85675 )
> Downloading DAYMET data for: bla at 40.22458/-88.85675 latitude/longitude !
> Done !

**Comment by khufkens** [comment]
Probably a windows issue, will try to resolve (migrating from downloader to curl)

**Comment by khufkens** [comment]
Ok This has been resolved now, and tested on Windows 7 (nothing higher however).

The package now uses the curl package (low level) instead of downloader(). In addition I figured out that the routine will time out if you are tunneling through a VPN or an aggressive firewall. This is the more likely reason for erros on downloading on locations within the US. I put in a new error message which is more granular and will tell you if it's a timeout issue.
